ow_regolith/include/DigStateMachine.h

```diff
@@ -0,0 +1,176 @@
+// The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+// Research and Simulation can be found in README.md in the root directory of
+// this repository.
+
+// Defines a finite state machine and associated states that estimate the phase
+// of a digging operation performed with the lander scoop.
+
+#ifndef DIG_FSM_H
+#define DIG_FSM_H
+
+#include <ros/ros.h>
+#include <tf/tf.h>
+
+#include <string>
+#include <memory>
+
+namespace ow_regolith {
+
+class DigStateMachine;
+
+class DigState
```

Comment:
I find it a bit odd that this class is top-level but all its subclasses are internal to DigStateMachine.  Suggest encapsulating it as well.

Reply:
Done.